pkg/runtime/k0s/runtime.go

```diff
@@ -15,185 +15,144 @@
 package k0s
 
 import (
+	"bytes"
 	"context"
 	"fmt"
 	"net"
-	"strings"
-	"time"
+	"os"
 
 	"github.com/sealerio/sealer/common"
+	containerruntime "github.com/sealerio/sealer/pkg/container-runtime"
+	"github.com/sealerio/sealer/pkg/infradriver"
 	"github.com/sealerio/sealer/pkg/registry"
 	"github.com/sealerio/sealer/pkg/runtime"
-	v2 "github.com/sealerio/sealer/types/api/v2"
-	"github.com/sealerio/sealer/utils/platform"
-	"github.com/sealerio/sealer/utils/ssh"
+	utilsnet "github.com/sealerio/sealer/utils/net"
 
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sync/errgroup"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtimeClient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // Runtime struct is the runtime interface for k0s
 type Runtime struct {
-	// cluster is sealer clusterFile
-	cluster   *v2.Cluster
-	Vlog      int
-	RegConfig *registry.Config
+	infra                infradriver.InfraDriver
+	Vlog                 int
```

Comment:
I don't know if I can use lowercase here？

Reply:
The Vlog now can't support debug mode well, but If we need to print some debug log, this would be effective.